build.sbt

```diff
@@ -106,6 +119,23 @@ lazy val `anorm-akka` = (project in file("akka"))
       "com.typesafe.akka" %% "akka-stream-contrib" % "0.6" % Test)
   )).dependsOn(anorm)
 
+lazy val pgVer = sys.env.get("POSTGRES_VERSION").getOrElse("9.4.1212")
+
+val jsonVer = Def.setting[String] {
```

Comment:
@cchantep can you explain this more? I see that play-json has a 2.6.x release for Scala 2.10, 2.11 and 2.12 so I think we should be able to use the same version for all of these versions of Scala.